### PR TITLE
feat(route): add curius.app

### PR DESCRIPTION
**Pull request by Ovler-Young**
## 该 PR 相关 Issue / Involved issue

无

## 完整路由地址 / Example for the proposed route(s)

```routes
/curius/links/yuu-yuu
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [x] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: docs/social-media.md [hunk 1]
```diff
@@ -344,6 +344,12 @@ Tiny Tiny RSS 会给所有 iframe 元素添加 `sandbox="allow-scripts"` 属性
 
 </Route>
 
+## Curis.app
+
+### 用户
+
+<Route author="Ovler-Young" example="/curis/yuu-yuu" path="/curis/:name" :paramsDesc="['用户名称，可在url中找到']"/>
```
Comment (TonyRL) [inline docs/social-media.md:351]: This example does not work.

Comment (TonyRL) [inline docs/social-media.md:351]: `path` here must match the path in `maintainer.js` **AND** `router.js`

File: lib/v2/curius/maintainer.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    '/link/:name?': ['Ovler-Young'],
```
Comment (TonyRL) [inline lib/v2/curius/maintainer.js:2]: The key here should be the same as `router.js` as well as `path` in the docs

File: lib/v2/curius/router.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+module.exports = function (router) {
+    router.get('/links/:name', require('./links'));
```
Comment (TonyRL) [inline lib/v2/curius/router.js:2]: The path here should be the same as `maintainer.js` and `path` in the docs

File: lib/v2/curius/radar.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'https://curius.app/': {
```
Comment (TonyRL) [inline lib/v2/curius/radar.js:2]:
```suggestion
    'curius.app': {
```
You need to specify the root domain only

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: lib/v2/curius/router.js (hunk 3 above)
Comment (Ovler-Young) [reply lib/v2/curius/router.js:2]: They're now the same.

File: docs/social-media.md (hunk 1 above)
Comment (Ovler-Young) [reply docs/social-media.md]: Fixed

Comment (Ovler-Young) [reply docs/social-media.md]: Fixed

File: lib/v2/curius/maintainer.js (hunk 2 above)
Comment (Ovler-Young) [reply lib/v2/curius/maintainer.js]: Fixed

File: lib/v2/curius/radar.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'curius.app': {
+        _name: 'Curius',
+        '.': [
+            {
+                title: 'Curius',
```
Comment (TonyRL) [inline lib/v2/curius/radar.js:6]:
```suggestion
                title: '用户',
```
`title` here should match H3 heading

File: lib/v2/curius/radar.js [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+                docs: 'https://docs.rsshub.app/social-media.html#curius',
+                source: '/*',
```
Comment (TonyRL) [inline lib/v2/curius/radar.js:8]:
```suggestion
                source: '/:name',
```

File: lib/v2/curius/radar.js [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+                target: '/*',
```
Comment (TonyRL) [inline lib/v2/curius/radar.js:9]:
```suggestion
                target: '/curius/links/:name',
```

File: lib/v2/curius/links.js [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+        url: `https://curius.app/api/users/${uid}/links?page=0`,
+        method: 'get',
+        headers: {
+            Referer: `https://curius.app/${username}`,
+        },
+    });
+
+    const items = response.data.userSaved.map((item) => ({
+        title: item.title,
+        description: `原文：${item.metadata.full_text.replace(/\n/gm, '<br/>')}${item.comments.length > 0 ? `<br/><br/>评论：${item.comments[0].text.substring(0, 100)}</br>` : ''}${item.highlights.length > 0 ? `<br/>标注：${item.highlights.map((highlight) => (highlight.comment ? `${highlight.highlight}<br/>评论：${highlight.comment.text}<br/>` : `<br/>标注：${highlight.highlight}<br/>`)).join('')}` : ''}`,
```
Comment (TonyRL) [inline lib/v2/curius/links.js:30]: Please use `art-template` for custom HTML content
docs: https://docs.rsshub.app/joinus/script-standard.html#template

File: lib/v2/curius/links.js [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+module.exports = async (ctx) => {
+    const username = ctx.params.name;
+
+    const name_response = await got({
+        url: `https://curius.app/api/users/${username}`,
+        method: 'get',
+        headers: {
+            Referer: `https://curius.app/${username}`,
+        },
+    });
```
Comment (TonyRL) [inline lib/v2/curius/links.js:13]: got request can be simplified
```suggestion
    const name_response = await got(`https://curius.app/api/users/${username}`, {
        headers: {
            Referer: `https://curius.app/${username}`,
        },
    });
```

File: lib/v2/curius/links.js [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+    const uid = data.user.id;
+    const name = `${data.user.firstName} ${data.user.lastName}`;
+
+    const response = await got({
+        url: `https://curius.app/api/users/${uid}/links?page=0`,
+        method: 'get',
+        headers: {
+            Referer: `https://curius.app/${username}`,
+        },
+    });
```
Comment (TonyRL) [inline lib/v2/curius/links.js:26]: got request can be simplified
```suggestion
    const response = await got(`https://curius.app/api/users/${uid}/links?page=0`, {
        headers: {
            Referer: `https://curius.app/${username}`,
        },
    });
```

**Pull request merged**